### PR TITLE
Add Units tests

### PR DESCRIPTION
**Pull request by sinarest1608**
Adds Unit test. Refer #5 for more details. 

To review:
Run the `tests/report_handler/test_report_handler.py` file and verify all tests passed. Please also check the flow of the code to see if any edge cases are missed. 

#10 Should be merged before this.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     // testing
     "python.testing.unittestArgs": [
         "-v",
+        "-s",
```
Comment (mbentz-uf) [inline .vscode/settings.json:30]: What does this flag do? Unless I comment it out VSCode doesn't find the tests.

**Review by mbentz-uf — CHANGES_REQUESTED** [review]
Locally I checkout out PR #10 and merged in the changes from this PR. Running the unit tests with:
- `python -m unittest -v`
- Using VSCode (after removing -s)

both result in a failed test:
> image
> image

File: .vscode/settings.json (hunk 1 above)
Comment (sinarest1608) [reply .vscode/settings.json:30]: Looks like it specifies the directory for tests. It should've been something like this:

image


I will fix the tests and add this change to the commit also.

**Comment by mbentz-uf** [comment]
Closing this PR because changes have been included in #10.
